source/docs/essentials/add-new-attribute.md

```diff
@@ -0,0 +1,156 @@
+---
+id: add-new-attribute
+title: Add a new attribute
+---
+
+Let's say we want to add a new element to display on a product's page (for instance, a product's rate). You will have to:
+
+1. Add the attribute's actual value to your database.
+2. Create a new module.
+3. Define the schema, implement the resolver and display it on screen.
+
+## Add a value to your database.
+
+The process to do so may vary with the backend software you are using. For Magento 2 (used in this example), you can refer to [this](https://www.fastcomet.com/tutorials/magento2/product-attributes).
+
+## Create a new GraphQL module
+
+We want to add a **User rate** feature to a product page. This feature will be stored in a new module called `mymodule` ; you could directly edit the files at `node_modules/front-commerce/path/to/something`, but this is deprecated.
+Learn more about Front-Commerce's folder structure [here](/docs/concepts/front-commerce-folder-structure.md).
```

Comment:
Rather than the folder structure, I would link to add a link to `/docs/essentials/extend-the-theme.html` which already explains in detail the module system (with overrides and all).